CTP/shell/src/com/navercorp/cubridqa/shell/main/JdbcLocalTest.java

```diff
@@ -45,47 +47,53 @@ public JdbcLocalTest(Context context){
 			context.setVersion(buildBit);
 		}
 		
-		log = new Log(CommonUtils.concatFile(context.getCurrentLogDir(), "test_jdbc.log"), false);
+		runCaseLog = new Log(CommonUtils.concatFile(context.getCurrentLogDir(), "run_case_details.log"), false);
 		this.context.setTestCaseRoot(com.navercorp.cubridqa.common.CommonUtils.translateVariable(this.context.getTestCaseRoot()));
 	}
 
 	private void start(){
 		String scenarioDir = this.context.getTestCaseRoot();
 		try {
 			if (scenarioDir == null || scenarioDir.length() <= 0) {
-				this.log.println("[ERROR] Scenario is not configured.");
```

Comment:
this.runCaseLog.println("[ERROR] Scenario is not configured.");

It's better to print the info to console.